subsys/bluetooth/shell/bt.c

```diff
@@ -123,6 +127,67 @@ static void print_le_addr(const char *desc, const bt_addr_le_t *addr)
 }
 #endif /* CONFIG_BT_CONN || (CONFIG_BT_BROADCASTER && CONFIG_BT_EXT_ADV) */
 
+#if defined(CONFIG_BT_TRANSMIT_POWER_CONTROL)
+static const char *txPowerFlag2str(int8_t flag)
```

Comment:
Sorry, I saw this already earlier, but forgot to comment on it: no camelCase in Zephyr, please - use underscore instead (not sure why you changed the style only for some specific functions?)

Reply:
I did so because I based it on the phy2str function, which isn't exactly camelCase but something in my mind thought if it had more words, it would be. I'll clean up what you commented :)